### PR TITLE
Sat image

### PR DESCRIPTION
**Pull request by JuliaWindmiller1987**
Creates a new file for plotting satellite images and other meteorological fields. Implements a function to retrieve and plot a satellite image via nasa worldview.

### CODE REVIEW

File: orcestra/flightplan.py [hunk 1]
```diff
@@ -389,3 +389,60 @@ def path_quickplot(path, sel_time, crossection=True):
...
+
+def calc_zonal_mean(field, lon_min, lon_max, lat_min, lat_max): 
+    import easygems.healpix as egh
+    
+    bbox = [lon_min, lon_max, lat_min, lat_max]
+    
+    if "lon"  in field.coords:
+        field = field.rename({'lon': 'longitude'})
+    if "lat"  in field.coords:
+        field = field.rename({'lat': 'latitude'})
```
Comment (d70-t) [inline orcestra/flightplan.py:401]: This probably isn't compatible with the latest version of easygems, which uses `lat` and `lon` for the coordinates by default.

File: orcestra/flightplan.py [hunk 2]
```diff
@@ -389,3 +389,60 @@ def path_quickplot(path, sel_time, crossection=True):
...
+    
+    field_lat = field_bbox.groupby(
+        field_bbox.latitude
+    ).mean(
+    )
+    return field_lat
+
+def find_edges(cwv, cwv_thresh, cwv_min = 0, lat_cwv_max = 9.0):
+    
+    import scipy.signal
```
Comment (d70-t) [inline orcestra/flightplan.py:417]: This line adds a dependency on `scipy`. Can you add this to `pyproject.toml` as well, such that `scipy` will be installed automatically when the `orcestra` package is installed.

**Review by d70-t — COMMENTED** [review]
`plot_sat` seems to be a quite straight forward addition :+1:

The addition of the find edges functions may be problematic (see comments) and seems to be accidentally part of this PR (it doesn't seem to belong to "Sat image").
I'd suggest to separate these out into it's own PR.

File: orcestra/flightplan.py (hunk 1 above)
Comment (JuliaWindmiller1987) [reply orcestra/flightplan.py:401]: I am a bit unclear on how to resolve this one... One option would be, I guess, to fix the minimum version of easygems you need in the .toml file and then use <lat> and <lon> but I am not sure if this is what we really want?

Comment (d70-t) [reply orcestra/flightplan.py:401]: Yes, I guess a minimum version of easygems would be the best option.

**Pull request closed without merging**